function.js

```diff
@@ -1,3 +1,14 @@
+function scrollingTop() {
+  if(window.scrollY!=0)
+  {
+      setTimeout(function() {
+         window.scrollTo(0,window.scrollY-30);
+          scrollingTop();
+      }, 0);
```

Comment:
I looked up, there is a requestAnimationFrame function you can potentially use.

Reply:
Oh thank you Ian. I can use that function.